frontend/app/routes/$lang+/_public+/apply+/$id+/confirmation.tsx

```diff
@@ -236,7 +236,12 @@ export default function ApplyFlowConfirm() {
       </button>
 
       <div className="mt-10 flex flex-wrap items-center gap-3">
-        <ButtonLink variant="primary" onClick={() => sessionStorage.removeItem('flow.state')} to="/apply">
+        <ButtonLink
```

Comment:
Is `size="lg"` not good? I don't think we have to add that much extran css classes.

Reply:
yeah so it was to align with the styles of the "print" button.  I've swapped out the native html button with the button component now

Reply:
also added the `action`